init-client.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
init-client should be a script provided by membrane-backend (eventually membrane-client)

Reply:
Removed altogether